splice/models.py

```diff
@@ -62,6 +62,13 @@ class AdgroupSite(db.Model):
     created_at = db.Column(db.DateTime(), nullable=False, default=datetime.utcnow)
 
 
+class BlacklistedIP(db.Model):
+    __tablename__ = "blacklisted_ips"
+
+    ip = db.Column(db.String(1024), nullable=False, primary_key=True)
```

Comment:
Not necessarily to be limited by 16 chars. AFAIK, a lot of vendors also offer CRID IP ranges, i.e. 192.168.100.0/24, instead of giving a very wide list of consecutive IPs.